skipper_test.go

```diff
@@ -109,6 +109,28 @@ func TestOptionsFilterRegistry(t *testing.T) {
 	assert.NotContains(t, fr, filters.BearerInjectorName)
 }
 
+func TestOptionsTracerInstanceOverridesOpenTracing(t *testing.T) {
+	tracer := &tracingtest.Tracer{}
+	o := Options{
+		Tracer:      tracer,
+		OpenTracing: []string{"noop"},
+	}
+
+	tr, err := o.tracerInstance()
+	assert.NoError(t, err)
+	assert.Same(t, tr, tracer)
```

Comment:
expected shoudl be the second argument, i.e. `assert.Same(t, tracer, tr)` - that's why I suggested to rename one ore the other